src/modules/claims/claims.types.ts

```diff
@@ -52,6 +52,7 @@ export interface Claim {
   acceptedBy?: string;
   isRejected?: boolean;
   namespace: string;
+  requestCompletionRedirectUrl?: string;
```

Comment:
Is this needed so that IKA can read the field?

Maybe simply the name as well
```suggestion
  redirectUri?: string;
```